src/main/java/jenkins/plugins/office365connector/model/Card.java

```diff
@@ -29,10 +30,10 @@
 
     private List<PotentialAction> potentialAction;
 
-    public Card(String summary, List<Section> sectionList) {
+    public Card(String summary, Section section) {
         this.summary = summary;
         this.themeColor = "3479BF";
-        this.sections = sectionList;
+        this.sections = Arrays.asList(section);
```

Comment:
just make sure here we have collection because this is what must be in JSON notification